### ISSUE TITLE
[FEATURE]: Volume level percentages

### ISSUE THREAD

**Issue by BLAndrew575** [XeroxDev/Loupedeck-plugin-VoiceMeeter#21]
### Consent

- [X] I have searched the existing issues and verified that there is no open issue for the same subject.
- [X] I understand that the Loupedeck VoiceMeeter plugin has NO affiliation with VoiceMeeter.com or VB-Audio

### Feature

I have been using voicemeter for a long time with my twitch stream for ease of use.  When I got my razer stream controller, I got it specifically so I could control voicemeters volume levels easily without having to pull up the program.  My only complaint is that you cant see what the volume levels are next to the knobs when changing the gain of the channels. So I have to have voicemeter open on a monitor at all times to monitor it. Instead it just shows gain and I cant label the knob.  I think it would be not only a huge benefit to add in the ability to see the volume percent, but also be able to name what the knob is controlling(Channel A, B, C, D and so on).  There is a similar feature in the volume controls general that comes with the stream deck, but unfortunately it doesnt actually control the outputting sounds to stream like the voicemeter specific plugin does.  

### Solution

Add label next to knob showing channel controlling and percentage volume is.
![Capture2](https://user-images.githubusercontent.com/106648858/224931999-eb94ae69-9537-4b4f-b799-d7545b0261fe.JPG)

**Comment by XeroxDev** [comment]
Hey and thank you for opening the issue.

This is a duplicate of #17 and was a bug in the last version. The latest version should already be on the marketplace, but to make sure: can you install the latest version of this plugin from the [release](https://github.com/XeroxDev/Loupedeck-plugin-VoiceMeeter/releases) tab and check if it works for you?

After testing the latest version, could you please respond back and let me know if it works or not (so I can check it out)

**Comment by XeroxDev** [comment]
Yeah. I've checked the version of the plugin in the marketplace. It's still 1.0.0 and should be 3.0.0 right now. I've already contacted loupedeck about it. Maybe something went wrong while publishing the latest version.

Please use the latest version from the release tab earlier and everything should be fixed.

**Comment by BLAndrew575** [comment]
Sorry I didnt get back to you sooner.  Streamer/content creator and have
been pushing to get a video out (every spare minute the last week)
Remembered today and got everything updated with the 3.0 version and it
works PERFECT.  Just what I needed! Thanks for the help! Truly appreciate
it! I just knew there had to be a way to get the volume percentage on the
labels without constantly trying to guess where it was XD  Thanks again!
